tests/frame/0069-frame.jsonld

```diff
@@ -0,0 +1,6 @@
+{
+  "@context": {
+    "ex": "http://example.org/vocab#"
```

Comment:
Need to include `"ex:info": {"@type": "@json"}` in the frame context for it to be used to compact the result.

Reply:
However, this results in an error in my implementation: " invalid frame: Invalid JSON-LD frame syntax; invalid value of `@type`: `@json`", which is likely either an implementation or a spec bug.